recipes/_configure_server.rb

```diff
@@ -45,7 +55,13 @@
 
     server_stanzas[stanza] = {}
     server_stanzas[stanza]['master_uri'] = master_uri
-    server_stanzas[stanza]['pass4SymmKey'] = pass unless pass.empty?
+
+    ruby_block 'set pass4SymmKey for individual cluster stanza' do
```

Comment:
You need to include the cluster name in the resource name otherwise resource cloning.